src/util/hashids.rs

```diff
@@ -1,7 +1,12 @@
 use harsh::Harsh;
 use std::sync::LazyLock;
 
-pub static HARSH: LazyLock<Harsh> = LazyLock::new(|| Harsh::builder().length(6).build().unwrap());
+pub static HARSH: LazyLock<Harsh> = LazyLock::new(|| {
+    Harsh::builder()
+        .length(6)
+        .build()
+        .expect("Harsh couldn't be build")
```

Comment:
See https://doc.rust-lang.org/std/result/enum.Result.html#recommended-message-style

Looking at the errors build can give (https://docs.rs/harsh/latest/harsh/enum.BuildError.html) a good message could be: "build with default alphabet and separator should succeed"